core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -4799,6 +4808,57 @@ && isHourKeyword(qc.getAst().token) && qc.getAst().paramCount == 1 && qc.getAst(
         return root;
     }
 
+
+    private boolean aliasAppearsInColsAndColArgs(QueryModel model, CharSequence alias) {
```

Comment:
What's the purpose of this function? Column args don't make a lot of sense. Shouldn't it be something like `aliasAppearsInColsAndFuncArgs`?

Reply:
Yes, I understand the purpose, but the method name isn't very clearn. Perhaps, a better name and a comment with a sample query around this method would help.